crowbar_framework/lib/crowbar/backup/restore.rb

```diff
@@ -285,7 +285,10 @@ def restore_database
       end
 
       def proposal?(file)
-        !file.basename.to_s.match(/(_network\.json$)|(^template-(.*).json$)|(^queue\.json$)/) && \
+        return false if file.basename.to_s =~ /^template-(.*).json$/
+        raw_data = JSON.parse(file.read)["raw_data"]
+        return false if raw_data.nil?
+        raw_data.key?("attributes") && raw_data.key?("deployment") && \
           file.to_s =~ /knife\/databags\/crowbar/
```

Comment:
Heh, I would have added that check with the template one to avoid loading the file, but +1.